### DISCUSSION TITLE
Application getting rejected

### DISCUSSION THREAD

**Discussion by Amsb303** [community/community#110928] [GitHub Education]
### Hi everyone,

I have tried to apply multiple times, and each one got rejected with a reason why. but my last two got rejected for no reason.
![Screenshot 2024-03-04 224106](https://github.com/community/community/assets/149975746/51c7c6c1-66f9-49a4-aff4-797ffb51b834)
![Screenshot 2024-03-04 224837](https://github.com/community/community/assets/149975746/7c5a8007-e1f8-4b78-9302-4b1dc2363f84)
![Screenshot 2024-03-04 224847](https://github.com/community/community/assets/149975746/bbf1a3d0-ee0d-45d6-b1e1-88884a9a830d)
![Screenshot 2024-03-04 224858](https://github.com/community/community/assets/149975746/2a8b996d-10c3-4666-9818-f1025054a3ac)

**Comment by rash522** [comment]
Just apply again by uploading a "Good" proof. and make sure your details are correct and clear (take that photo by logging your phone and select back camera, otherwise it wouldn't be clear). and it should be issued by your college/institute.

note: Don't use VPN while you apply for it. and don't select wrong college. if your college from outside of your area, just contact and get support from them.

Try again and Good luck!
